<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasedBlockWindow</class>
 <widget class="QMainWindow" name="BasedBlockWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Text label:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Info</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Script:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Name Block:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Text lineEdit:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="4" column="3">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>LineEdit</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Label</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="4">
     <widget class="QTextEdit" name="textEdit">
      <property name="tabStopWidth">
       <number>30</number>
      </property>
     </widget>
    </item>
    <item row="5" column="3" colspan="2">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Test Script</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="0" column="4" rowspan="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>NULL</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Set Image</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Delete Image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="diagramscene.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Block</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="diagramscene.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Block</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="diagramscene.qrc"/>
 </resources>
 <connections/>
</ui>
